### DISCUSSION TITLE
Database Schema

### DISCUSSION THREAD

**Discussion by kmohamed1000** [jobrunr/jobrunr#121] [Q&A]
Is there an option to define a specific db schema for Storage.

i.e I would like to create the table parts_master.jobrunr_migrations, how do I tell jobrunr to use the parts_master schema?

https://www.jobrunr.io/en/documentation/installation/storage/

**Comment by rdehuyss** [comment]
How are you currently configuring the StorageProvider? Or are you using the spring-boot-starter integration?

You could setup a separate DataSource (including the schema then in the connection string) and then pass that to the StorageProvider. Does that make sense?

**Comment by rdehuyss** [comment]
This may indeed be an issue. See #136 .

**Comment by rdehuyss** [comment]
Hi @kmohamed1000 , are you able to build and test a version of JobRunr which supports different schema's before I release a new version?
